static/styles/settings.css

```diff
@@ -344,6 +344,10 @@ input[type=checkbox] + .inline-block {
     margin-top: 10px;
 }
 
+.organization-settings-parent {
+    width: 98%;
+}
```

Comment:
Why did you make this change?  Seems like it might impact other parts of the page.